client/lib/performance-tracking/lib.js

```diff
@@ -47,9 +46,7 @@ const buildMetadataCollector = ( metadata = {} ) => {
 };
 
 const isPerformanceTrackingEnabled = () => {
-	const isEnabledForEnvironment = config.isEnabled( CONFIG_NAME );
-	const isEnabledForCurrentInteraction = abtest( AB_NAME ) === AB_VARIATION_ON;
-	return isEnabledForEnvironment && isEnabledForCurrentInteraction;
+	return config.isEnabled( CONFIG_NAME );
```

Comment:
I'd consider inlining this constant. It's used only once.

Reply:
Make sense, inlining